definitions/divorce/json/CaseRoles.json

```diff
@@ -6,6 +6,13 @@
     "Name": "Respondent's Solicitor",
     "Description": "Role to isolate events available for Respondent's Solicitor"
   },
+  {
+    "LiveFrom": "01/01/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "[PETSOLICITOR]",
```

Comment:
This case role already exists :)

Reply:
Pardon, i had added this here as a placeholder for `caseworker-caa`, and had forgotten to change it by the time i pushed